docs/guest.md

```diff
@@ -26,8 +26,8 @@ The Data Manager Virtual Machine also needs to be installed to enable TKGS backu
 
 ### Install Object Storage Plugin
 
-Volume backups are stored in an object store bucket, e.g., S3. Currently these are stored in the same bucket configured for the object storage plugin of Velero. Before installing the vSphere plugin, please install and configure the object storage plugin.
-Here is an example, [velero-plugin-for-aws](https://github.com/vmware-tanzu/velero-plugin-for-aws/blob/master/README.md).
+Volume backups are stored in an object store bucket, e.g., AWS S3. They are stored in the same bucket configured for the object storage plugin of Velero. Before installing the vSphere plugin, a Velero object storage plugin is required. Currently, only AWS plugin is supported and compatible with vSphere plugin.
```

Comment:
Should we mention Minio somewhere?  Otherwise folks may think they can only use AWS S3.

Reply:
How about we can mention here that s3-compatible object stores can be supported via AWS plugin? Minio is just one of s3-compatible object store

Reply:
Yes, that sounds good.